force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -102,18 +129,19 @@ public with sharing class ClassesObjectsMethods {
      */
     public static Integer countVowels(String str) {
         // Initialize the result Integer
-
-        /*
-        for () {
-            // get individual characters from the string - hint use substring or split
-            // check if the character is a vowel
-            // if it is a vowel, increment the result Integer
+        Integer numberOfVowels = 0;
+        String vowels = 'aeiou';
+        List<String> charList = new List<String>();
+        for (Integer i = 0; i < str.length(); i++) {
+            charList.add(str.substring(i, i + 1));
```

Comment:
We can skip this loop (and reduce lines of code) by using the `split()` method you employed in the previous method.
```suggestion
        List<String> charList = str.split(' ');
```

Reply:
Or, another way to approach this is to keep the loop and check here if each character is a vowel (there's no need for 2 loops):
```suggestion
    List<String> charList = new List<String>();
    for (Integer i = 0; i < str.length(); i++) {
        String letter = str.substring(i, i+1).toLowerCase();
        if (vowels.contains(letter)) {
            numberOfVowels++;
        }
```